cfme/services/catalogs/orchestration_template.py

```diff
@@ -207,7 +187,38 @@ def exists(self):
             return False
 
 
-@navigator.register(OrchestrationTemplate, 'All')
+@attr.s
+class OrchestrationTemplateCollection(BaseCollection):
+    """A collection for the :py:class:`cfme.services.catalogs.orchestration_template`"""
+    ENTITY = OrchestrationTemplate
+
+    def instantiate(self, template_type, template_name, description, draft, content):
+        return self.ENTITY.from_collection(self, template_type, template_name,
+                                           description, draft, content)
+
+    def create(self, template_name, description, template_type, draft=None, content=None):
+        self.template_type = template_type
+        view = navigate_to(self, 'AddTemplate')
+        if template_type == "CloudFormation Templates":
+            temp_type = "Amazon CloudFormation"
+        elif template_type == "Heat Templates":
+            temp_type = "OpenStack Heat"
+        elif template_type == "Azure Templates":
+            temp_type = "Microsoft Azure"
```

Comment:
*REQUIRED**
Please make a dictionary instead of `if elif`

Reply:
Pulling it from yamls now